sql/2020/third-parties/distribution_of_size_and_time_by_third_parties.sql

```diff
@@ -39,7 +39,7 @@ SELECT
   category,
   percentile,
   APPROX_QUANTILES(body_size, 1000)[OFFSET(percentile * 10)] AS body_size,
-  APPROX_QUANTILES(time, 1000)[OFFSET(percentile * 10)] AS time -- noqa: L010
```

Comment:
All keywords capitalized.
This noqa doesn't seems to impact anything.

Reply:
Will do. Looks like until 0.9.2 `time` was considered a keyword. Think I fixed it with https://github.com/sqlfluff/sqlfluff/pull/2283

Reply:
I've fixed it.